content/initiatives/automated-demand-response.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Automated Demand Reponse"
+slug = "automated-demand-response"
+template = "initiatives/initiative.html"
+
+[extra]
+funders = [
+ "tweedegolf",   
+ "elaadnl",
+]
+
+blogposts = [
+]
++++
+
+### Automated Demand Response
+
+With the rise of renewable energy resources and ongoing electrification, automated demand response plays a crucial role in optimizing the use of our electricity grid. 
+
+This project is building a modern, open-source Rust implementation of OpenADR 3.0, the latest version of OpenADR, the automated demand response standard that's being developed by the [OpenADR Alliance](https://www.openadr.org/).
+
+Written in [Rust](https://www.rust-lang.org/), our implementation is fast, lightweight, reliable, and can be used in many different environments. It will be an important building block for a multitude of digital products and services required for the further electrification of our energy systems and the energy transition.
+
+### What We've Done
+
+[Tweede golf](https://tweedegolf.nl/en) started development in June 2024 with initial funding from [ElaadNL](https://elaad.nl/en/), the Knowledge & Innovation center for the joint Dutch grid operators that researches and tests smart and sustainable charging of electric vehicles.
+
+To enable global collaboration and ensure neutral governance, in October 2024 the project became part of the [Linux Energy Foundation OpenLEADR](https://lfenergy.org/projects/openleadr/) project, that was already hosting an OpenADR 2.0b implementation written in Python.
+
+The reponsitory for the OpenADR 3.0 Rust implementation is called [openleadr-rs](https://github.com/OpenLEADR/openleadr-rs).
+
+### Current Status
+
+Currently (November 2024) the project is seeing its first external contributions and is almost ready to do a first field-test-ready release, for which we're seeking funding.
+
+### What's Next
+
+After inital funding by ElaadNL and Tweede golf in 2024, the project is currently (November 2204) **seeking a total of EUR 40.000 in funding** for:
+
+**In Dec 2024 / Jan 2025:**
++ Doing a first field-test-ready release 
+
+**During 2025:**
++ Ongoing development and maintenance
++ Continued community building and contributor support
++ Additional releases when new (minor) versions of the OpenADR starndard are published
++ Preparing for a first long term support release
+
+If you are interested in funding this work, please get in touch with us via <a href="mailto:donate@trifectatech.org">donate@trifectatech.org</a> and request the project's sponsorship prospectus.
```

Comment:
What if we make this a direct download? (This is a suggestion for all projects that we have a prospectus for.)

Reply:
Maybe in the future, but for now we keep the prospectus private; for prospects only